lib/colorado_lottery.rb

```diff
@@ -0,0 +1,40 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants,
+              :name
```

Comment:
you don't have a `name`instance variable, so i wouldn't expect to see this attr_reader here